packages/executor/src/index.ts

```diff
@@ -140,8 +140,18 @@ export class ChugSplashExecutor extends BaseServiceV2<
       executorAddresses.push(w.address)
     }
 
+    const executors = await Promise.all(
+      this.state.keys.map(async (el) =>
+        new ethers.Wallet(el.privateKey).getAddress()
+      )
+    )
```

Comment:
The Promise is unnecessary since you can just do `wallet.address` instead of `await wallet.getAddress()`